.eslintrc.cjs

```diff
@@ -16,5 +16,6 @@ module.exports = {
       'warn',
       { allowConstantExport: true },
     ],
+    "react/prop-types": "off"
```

Comment:
Why this?

Reply:
when i added props to newly added components (e.g. `handleClickTaskAdd` in `Form.jsx`), then there appeared an eslint error saying that these props have `any` type

now, i could use PropTypes and specify the types as requested, but the thing is that this is not really used anymore. if you want to set types strictly, you should probably use TypeScript for that.

but that's just an opinion though

Reply:
that's weird. i don't see the error anymore when i comment it off. i also don't recall exactly what was the message.
i guess i just figured out to disable PropTypes all together 🤷‍♂️